### DISCUSSION TITLE
Microsoft Teams Snippet Integration

### DISCUSSION THREAD

**Discussion by burghoffdavid** [GitbookIO/community#539] [Feature Requests]
While a full Microsoft Teams / Office 365 suite Integration (see [here](https://github.com/orgs/GitbookIO/discussions/116)) might be a big undertaking, supporting the `@GitBook save` Snippet (see Slack Integration) functionality via a Microsoft Teams Integration might be a good start and easier achievable. 

Cheers,
David

**Comment by milothiesen** [comment]
we would really really like this as well. We are a teams org and so much knowledge is shared in various chats, it'd be great to capture the chats as snippets.

**Comment by shlomof-pyle** [comment]
this would really make a huge impact for my org, to take this from useful little tool, to something foundational
